SQL/Archive/15.10/2016-03-15_ReqAcctSiteField.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE users ADD site varchar(255);
```

Comment:
@samirdas "There is already a CenterID field"
@driusan: Why is this patch in 15.10? And if there is a change, where's the update to the default schema?